src/Methods/Passable.php

```diff
@@ -189,14 +187,7 @@ function (PassableContract $passable) use ($originalClassReflection) {
             );
 
         if ($result = $this->hasFound()) {
-            $methodReflection = $this->getMethodReflection();
-            if (get_class($methodReflection) === PhpMethodReflection::class) {
-                $methodReflection = Mockery::mock($methodReflection);
-                $methodReflection->shouldReceive('isStatic')
-                    ->andReturn($this->isStaticAllowed());
-            }
-
-            $this->setMethodReflection($methodReflection);
+            $this->setMethodReflection($this->getMethodReflection());
```

Comment:
And tests are passing? If yes, why I was using it in the first place? 🧐

Reply:
Yeah I just wanted to see what is failing. Then I'll try to find a way to fix it.